src/test/scripts/get_product_lts_version.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import urllib.request
+import json
+
+known_lts_version = {
+	'jira-software':'8.13.7',
+	'confluence':'7.12.2',
+	'stash':'7.12.1',
+}
+
+def get_lts_version(argv):
+	lts_version=''
+	product = argv[0].lower()
+	if product == 'jira':
+		product = 'jira-software'
+	elif product == 'bitbucket' :
+		product = 'stash'
+
+	if product in known_lts_version:
+		url = f"https://my.atlassian.com/download/feeds/archived/{product}.json"
```

Comment:
Just looking at the raw data from these respective product URL's, from what they present I dont see how the LTS version can be calculated based on the code that parses it below?

Reply:
Check edition field (Standard/None means none LTS and Enterprise is LTS version)